integration/node/features/support/encrypt.js

```diff
@@ -6,9 +6,8 @@ const { Given, When, Then } = require('@cucumber/cucumber')
 const assert = require('assert');
 const asherah = require("asherah");
 const fs = require("fs");
-const os = require("os");
 
-const fileDirectory = os.tmpdir() + "/";
+const fileDirectory = "/tmp/";
```

Comment:
why?

Reply:
This is the working directory for all cross-language tests in the suite. For example, asherah-go encrypts a test payload and writes it to a file in `/tmp` which is then loaded and decrypted by the asherah-node package (and vice versa).

As it is now, asherah-node's encryption output is not found by the downstream tests in this expected location, resulting in the errors described in #120


See:
asherah-node decrypt: https://github.com/godaddy/asherah-node/blob/36934f10d6ce562e770db0300252d731193b557c/integration/node/features/support/decrypt.js#L8

asherah-go: https://github.com/godaddy/asherah/blob/main/tests/cross-language/go/const.go#L10